Keyboards/KeyboardsBase/KeyboardKeys.swift

```diff
@@ -291,25 +283,46 @@ class KeyboardKey: UIButton {
     } else if DeviceType.isPad {
       adjustPadKeyWidth()
     }
+  }
 
+  /// Adjusts the style of the button based on different states
+  func adjustButtonStyle() {
     guard let isSpecial = layer.value(forKey: "isSpecial") as? Bool else { return }
 
-    if key == "shift" {
-      // Switch the shift key icon given its state.
+    switch key {
+    case SpecialKeys.indent:
+      backgroundColor = specialKeyColor
+
+    case SpecialKeys.capsLock:
+      switch capsLockButtonState {
+
+      case .normal:
+        backgroundColor = specialKeyColor
+        styleIconBtn(btn: self, color: UIColor.label, iconName: "capslock")
+
+      case .locked:
+        backgroundColor = keyPressedColor
+        styleIconBtn(btn: self, color: UIColor.label, iconName: "capslock.fill")
+      }
+
+    case "shift":
       if shiftButtonState == .shift {
         backgroundColor = keyPressedColor
         styleIconBtn(btn: self, color: UIColor.label, iconName: "shift.fill")
-      } else if shiftButtonState == .caps {
-        backgroundColor = keyPressedColor
-        styleIconBtn(btn: self, color: UIColor.label, iconName: "capslock.fill")
```

Comment:
Losing this line is disabling an indication that we're in caps lock on iPhones :)